tests/test_batch_run.py

```diff
@@ -85,6 +85,7 @@ def get_local_model_param(self):
     def step(self):
         self.datacollector.collect(self)
         self.schedule.step()
+        self._advance_time()
```

Comment:
Shouldn’t `self.schedule.step()` take care of this now? (also in the other tests)

Reply:
Good catch.